server/core/src/main/java/org/infinispan/server/core/transport/NettyTransport.java

```diff
@@ -71,6 +75,7 @@
 
    //-1 if not set
    private int nettyPort = -1;
+   private boolean shouldShutdownIOGroup;
```

Comment:
nit:
the value of `shouldShutdownIOGroup` is always `cacheManager == null`. IMO, the field is no required.

Reply:
Let me take a look again. There was something with the single port router where this is null, let me check again.

Reply:
oh gotcha, you mean just check cacheManager, sure.